types/node/test/crypto.ts

```diff
@@ -1336,6 +1336,32 @@ import { promisify } from 'node:util';
     aliceSecret === bobSecret;
 }
 
+{
+    const alice = crypto.createECDH('prime256v1');
+    const bob = crypto.createECDH('prime256v1');
+
+    alice.setPrivateKey('abcd', 'hex');
+    bob.setPrivateKey(Buffer.from('abcd', 'hex'));
+
+    alice.generateKeys();
+
+    let alicePublicKey = alice.getPublicKey(); // $ExpectType Buffer
+    alicePublicKey = alice.getPublicKey(null);
```

Comment:
There did not appear to be an existing `createECDH` test so I added one, but these 3 lines are the actually impacted code.